flutter/rsvp/lib/Models/event.dart

```diff
@@ -3,34 +3,43 @@ class Event {
   String _name;
   String _date; // Remove it later. Just use _dateTime
   DateTime _dateTime;
+  String _description;
   String _creator;
   String _location;
+  int _minNum;
   bool _status;
 
   Event(this._name, this._date); // Remove Later
 
-  Event_Constructor(String link, String name, String creator, DateTime dateTime, String location, bool status) {
```

Comment:
I think we should keep this as DateTime, it shouldn't be a problem and isn't difficult to create one.

Reply:
ok! then, lets keep this! :)